apps/web/src/pages/index.astro

```diff
@@ -5,10 +5,35 @@ import Logo from "../components/logo.astro";
 import Link from "../components/link.astro";
 import Footer from "../components/footer.astro";
 ---
+<style>
```

Comment:
I usually move style tags to the top bc my smooth brain forgets there are styles hidden at the bottom. But I could move it back if you want. I only moved bc I got snagged bc didn't realize they were down below and styles were getting overriden.